src/screens/ForgotPassword/handler.ts

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+
+import { AxiosResponse } from 'axios'
+
+import AuthAdapter from 'adapters/auth'
+import { allFieldsFilled } from 'helpers/formValidation'
+
+type SubmitHandler = {
+  requestSuccess: boolean
+  error: string
+  handleSubmit: (e: React.SyntheticEvent) => void
+}
+
+const SubmitHandler = (): SubmitHandler => {
+  const [requestSuccess, setRequestSuccess] = useState(false)
+  const [error, setError] = useState('')
+
+  const handleSubmit = async (e: React.SyntheticEvent) => {
+    e.preventDefault()
+
+    const target = e.target as typeof e.target & {
+      email: { value: string }
+    }
+
+    if (!allFieldsFilled([target.email.value])) {
+      setError('Email is required')
+      setRequestSuccess(false)
```

Comment:
`requestSuccess` is still `false` at this point so I guess it isn't needed here? 🤔

Reply:
Because forgot password doesn't redirect to any page if the request success so when user successfully performs their request then `requestSuccess` will become to true and success alert will be displayed and then if user unsuccessfully perform their request then now success alert and error alert will displayed at the same time if I remove `setRequestSuccess(false)`

![Screen Shot 2564-06-22 at 12 30 46](https://user-images.githubusercontent.com/29707647/122868811-ad69b900-d355-11eb-84c8-da991896a3d4.png)